.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This one doesn't belong here... please remove. Not sure if it's worth updating gitignore with this...